hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/DirectInStoreFlushContext.java

```diff
@@ -33,11 +33,11 @@
  * To be used only when PersistedStoreEngine is configured as the StoreEngine implementation.
  */
 @InterfaceAudience.Private
-public class PersistedStoreFlushContext extends DefaultStoreFlushContext {
+public class DirectInStoreFlushContext extends DefaultStoreFlushContext {
```

Comment:
Why the change in naming? 'DirectInStore' seems a bit confusing... if you want to change it, perhaps just 'DirectStore'?

Reply:
Name change suggestion came after comments from @saintstack on HBASE-25391.

Reply:
I feel like the "In" is a bit confusing. Originally everything was Persisted due to the feature of persisting the SFM, but I get the confusion for some of the internal classes. "Direct" should be fine